apps/bridge/src/components/DepositContainer/DepositContainer.tsx

```diff
@@ -54,6 +54,10 @@ export function DepositContainer() {
   const { data: readERC20Approval, error: readERC20ApprovalError } = useIsContractApproved({
     contactAddress: selectedAsset.L1contract,
     address,
+    spender:
+      selectedAsset.protocol === 'CCTP'
+        ? publicRuntimeConfig.l1CCTPTokenMessengerAddress
+        : publicRuntimeConfig.l1BridgeProxyAddress,
```

Comment:
Maybe extract a sort of `getSpender` function since this logic seems to be used twice

Reply:
updated, added a const